api/licenses/serializers.py

```diff
@@ -16,6 +16,7 @@ class LicenseSerializer(JSONAPISerializer):
     type = TypeField()
     name = ser.CharField(required=True, help_text='License name')
     text = ser.CharField(required=True, help_text='Full text of the license')
+    required_properties = ser.ListField(source='properties', read_only=True, help_text='Required attributes for the license')
```

Comment:
I think `required_fields` would be better. `help_text='Fields required for this license to help front-end validators'` or similar.